interfaces/builtin/x11.go

```diff
@@ -83,6 +83,11 @@ unix (bind, listen, accept)
 # For Xorg to detect screens
 /sys/devices/pci**/boot_vga r,
 /sys/devices/pci**/resources r,
+
+# Allow writing an Xwayland Xauth file
+# (see https://gitlab.gnome.org/GNOME/mutter/merge_requests/626)
+/run/user/[0-9]*/.mutter-Xwaylandauth.* rw,
+/run/user/[0-9]*/mutter/Xauthority rw,
```

Comment:
I would consider leaving this part out until we have a better understanding of who would use those permissions: no snaps currently depend on that access, and the slot doesn't currently grant write access to other Xauthority paths (`$XDG_RUNTIIME_DIR/.Xauthority` and `~/.Xauthority` are the common cases.

We're not at a stage where you'd be able to run gnome-shell with `XDG_RUNTIME_DIR` pointing at the usual location, so it is not clear these rules on their own would be of any value.

Reply:
Thanks for the feedback, I will just remove the rules in this PR then and leave a TODO in the x11 slot interface about providing access to something like XWayland creating an Xauth file

Reply:
This is done, the PR only removes the wayland slot rules and adds a TODO to the x11 slot about this